src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -819,13 +1117,17 @@
                     return false;
                 }
             }
-            else
+            else if (refKindCompareMode == RefKindCompareMode.RefMatchesOutInRefReadonly)
```

Comment:
>refKindCompareMode [](http://example.com/codeflow?start=21&length=18)

Since this method is static, perhaps we should assert our assumptions about ```refKindCompareMode``` values